examples/multiple-daemons/dataflow.yml

```
@@ -0,0 +1,37 @@
+communication:
+  zenoh:
+    prefix: example-multiple-daemons
+
+daemon_config: Tcp # or Shmem
```

Aesthetically speaking:
- `daemon_config` should be included within the communication block as it also concern communication. 
- `daemon_config` might not mean much for a lot of people. Can we maybe rename this: `inter_process_com`, `IPC` or `process_to_process` as it might be more explicit than `daemon_config` .
- it would be good to show that `zenoh` communication concern between machine communication and not between process. I would encapsulate it within a `machine_to_machine` block


```suggestion
communication:
  machine_to_machine:
    zenoh:
      prefix: example-multiple-daemons
  process_to_process: Tcp # or Shmem
```

---

Implemented in d6571bf. Instead of `machine_to_machine` and `process_to_process` I used `remote` and `local` as keys. Is this ok with you?

---

Since we still don't support zenoh, I removed zenoh from the configuration options for now. The previous keys are still accepted, but ignored (with a warning). In order to avoid confusing users, I removed the daemon and zenoh config from our dataflow examples, templates, and docs in 72a57ce .

---

Can we maybe give an example of how to use the `remote` and `local` keys?

---

See the `dataflow.yml` of the new `multiple-daemons` example. For now we marked the new config keys as unstable (see my comment below).